Code/Framework/AzManipulatorTestFramework/Source/DirectManipulatorViewportInteraction.cpp

```diff
@@ -118,10 +118,11 @@ namespace AzManipulatorTestFramework
         return m_manipulatorManager->Interacting();
     }
 
-    DirectCallManipulatorViewportInteraction::DirectCallManipulatorViewportInteraction()
+    DirectCallManipulatorViewportInteraction::DirectCallManipulatorViewportInteraction(
+        AZStd::shared_ptr<AzFramework::DebugDisplayRequests> debugDisplayRequests)
         : m_customManager(
               AZStd::make_unique<CustomManipulatorManager>(AzToolsFramework::ManipulatorManagerId(AZ::Crc32("TestManipulatorManagerId"))))
-        , m_viewportInteraction(AZStd::make_unique<ViewportInteraction>())
+        , m_viewportInteraction(AZStd::make_unique<ViewportInteraction>(AZStd::move(debugDisplayRequests)))
```

Comment:
question: you're moving a shared_ptr into a unique_ptr, is there any reason why the constructor param cannot also be a unique_ptr?

Reply:
I'm not moving a `shared_ptr` into a `unique_ptr` I'm passing a `shared_ptr` as an argument to the `ViewportInteraction` constructor, they are totally separate (it needs to be a `shared_ptr` for the DebugDisplayBus to the caller can safely hold onto a reference to the type and use it to query state)

Reply:
Ah, you're right! my bad

Reply:
No worries, hard to see in the diff for sure